lib/utils/sthUtils.js

```diff
@@ -153,68 +153,77 @@ function getOriginEnd(aDate, resolution) {
  * @return {Number} Returns the offset
  */
 function getOffset(resolution, date) {
-  var offset;
-  switch (resolution) {
-    case sthConfig.RESOLUTION.SECOND:
-      offset = date.getUTCSeconds();
-      break;
-    case sthConfig.RESOLUTION.MINUTE:
-      offset = date.getUTCMinutes();
-      break;
-    case sthConfig.RESOLUTION.HOUR:
-      offset = date.getUTCHours();
-      break;
-    case sthConfig.RESOLUTION.DAY:
-      offset = date.getUTCDate();
-      break;
-    case sthConfig.RESOLUTION.MONTH:
-      offset = date.getUTCMonth() + 1;
-      break;
-  }
-  return offset;
+    var offset;
+    switch (resolution) {
+        case sthConfig.RESOLUTION.SECOND:
+            offset = date.getUTCSeconds();
+            break;
+        case sthConfig.RESOLUTION.MINUTE:
+            offset = date.getUTCMinutes();
+            break;
+        case sthConfig.RESOLUTION.HOUR:
+            offset = date.getUTCHours();
+            break;
+        case sthConfig.RESOLUTION.DAY:
+            offset = date.getUTCDate();
+            break;
+        case sthConfig.RESOLUTION.MONTH:
+            offset = date.getUTCMonth() + 1;
+            break;
+    }
+    return offset;
 }
 
 /**
  * Returns a date in RFC 3339 format
  * @return {string} The formatted date
  */
 function getISODateString(date) {
-  function pad(n, isMs) {
-    var result = n;
-    if (!isMs) {
-      result = (n < 10) ? '0' + n : n;
-    } else if (n < 10) {
-      result = '00' + n;
-    } else if (n < 100) {
-      result = '0' + n;
+    function pad(n, isMs) {
+        var result = n;
+        if (!isMs) {
+            result = n < 10 ? '0' + n : n;
+        } else if (n < 10) {
+            result = '00' + n;
+        } else if (n < 100) {
+            result = '0' + n;
+        }
+        return result;
     }
-    return result;
-  }
 
-  return date.getUTCFullYear() + '-' +
-    pad(date.getUTCMonth() + 1) + '-' +
-    pad(date.getUTCDate()) + 'T' +
-    pad(date.getUTCHours()) + ':' +
-    pad(date.getUTCMinutes()) + ':' +
-    pad(date.getUTCSeconds()) + '.' +
-    pad(date.getUTCMilliseconds(), true) + 'Z';
+    return (
```

Comment:
This is another case in which excesive line break makes the code worse.

Reply:
Fixed 02ca383